git/gogit/client.go

```diff
@@ -418,7 +418,7 @@ func (g *Client) Push(ctx context.Context, cfg repository.PushConfig) error {
 		ProxyOptions: g.proxy,
 		Options:      cfg.Options,
 	})
-	return goGitError(err)
+	return err
```

Comment:
Please add context to this error using `fmt.Errorf`.

Reply:
Done, thx for the quick review!